src/main/resources/db/migration/V6.3.0.2.sql

```diff
@@ -0,0 +1,23 @@
+-- Dataset types have been added. See #10517 and #10694
+--
+-- Insert the default dataset type: dataset.
+INSERT INTO datasettype (name) VALUES ('dataset');
```

Comment:
In a previous version of this SQL script, I was adding "software" and "workflow" but @scolapasta and @qqmyers think we should avoid this.

Please see 200a45a261 for how I adjusted this SQL script as well as code, tests, and docs that assumed that "software" and workflow would be present.

Reply:
I mentioned this in standup and there were no objections from @qqmyers or @scolapasta. Resolving.